examples/text-generation/utils.py

```diff
@@ -368,13 +365,25 @@ def initialize_model(args, logger):
         "revision": args.model_revision,
         "token": args.token,
     }
+    if args.disk_offload:
+        model_kwargs["device_map"] = "auto"
+        model_kwargs["offload_folder"] = "/tmp/offload_folder/"
```

Comment:
does it work if we don't provide offload_folder (as anyway it does not write to it) ? alternatively, can we use the same path we use for const serialization to simplify the configuration?